packages/core/core/src/Strapi.ts

```diff
@@ -403,6 +406,7 @@ class Strapi extends Container implements Core.Strapi {
 
     this.startupLogger.logStartupMessage({ isInitialized });
 
+    this.log.info('Strapi server started successfully');
```

Comment:
isn't the logStartupMessage already doing this (similar at least)

Reply:
Yeah, but that always goes to console and bypasses the strapi logger

Reply:
I'm not sure why we do that though, do you know the reason? It seems like we could pass the logger to startupLogger and it could use that

Reply:
hum it's doing "beautiful" logs which means it cannot be put in a json but the logger needs to be standardized.

For the simple log we could pass the logger just not for the boxen thingy

Reply:
So would it make sense to keep this "simple" notification but move it to the startupLogger?

Reply:
or just remove that one line from the startupLogger and keep it here to separate concerns & do less work :)

Reply:
Ohh, wait. No, startupLogger doesn't even have anything that says Strapi has started. It's just the big box, and then a "welcome back" but otherwise nothing. That's why I'm adding this

Reply:
ok just sounds strange. will need to revamp that sooner or later just let's keep what you did